lib/rules/at-rule-empty-line-before/__tests__/index.mjs

```diff
@@ -754,6 +754,20 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: ['always', { ignoreAtRules: /^--my-/ }],
```

Comment:
I've added simple tests for regex without string since string-wrapped regex tests already exist:

https://github.com/stylelint/stylelint/blob/aed00cd75ecc3665e776f8d5d974c83d24212483/lib/rules/at-rule-empty-line-before/__tests__/index.mjs#L757-L795.